lib/applicants/field-resolvers.ts

```diff
@@ -42,3 +42,43 @@ export const applicantMedicalInformationResolver: Resolver<Applicant> = async (
     where: { applicantId: parent?.id },
   });
 };
+
+/**
+ * Field resolver to fetch the medical history object associated with an applicant
+ * @returns MedicalHistory obejct
```

Comment:
```suggestion
 * Field resolver to fetch the medical history associated with an applicant, including the physician and application ID for every completed application. The physician data is current (not from the time of the application).
 * @returns Array of medical history records
```
Please be more descriptive with comments. What is a medical history object? Also, this API returns an array of records that contain physician data and application ID. This information would be very helpful to someone trying to use this API or understand what it does.